src/main/java/dropwizard/springboot/dropwizard/to/springboot/controller/RestController.java

```diff
@@ -15,4 +27,50 @@ public class RestController {
     public String displayName(@RequestParam(name="name", required = false, defaultValue = "Stranger") String name){
         return "Hello! "+name;
     }
+
+    /**
+     * Endpoint to create/save a user
+     * @param userDto, takes userDto fields required to create and save user
+     * @return returns the entity of saved user
+     */
+    @PostMapping("/saveUser")

```

Comment:
If your using `@Post` then ~~save~~ is not required, as Post means saving a user